components/collection/unlockable/CountdownTimer.vue

```diff
@@ -24,6 +24,7 @@ import { useCountDown } from './utils/useCountDown'
 const animation = { duration: 30000, easing: (t) => t }
 const getPositiveNumber = (number: number) => Math.max(0, number)
 
+const slidesArray = Array(slidesCountOnTimeCountdown).fill(0)
```

Comment:
```suggestion
const slidesArray = ref(Array(slidesCountOnTimeCountdown).fill(0))
```
?

Reply:
Why should we make it a ref. it is a constant value.

Reply:
alright though it was common good pratice